content/en/examples/security/kind-with-cluster-level-baseline-pod-security.sh

```diff
@@ -67,4 +67,4 @@ spec:
       ports:
         - containerPort: 80
 EOF
-kubectl apply -f /tmp/pss/nginx-pod.yaml
+kubectl apply -f -
```

Comment:
Any reason you piped the manifest in here rather than using the same `example-baseline-pod.yaml`?

Reply:
The _original_ motivation for this was, I think, to help people doing a demo. Conference wifi!